src/adapter/src/coord/sequencer/inner/peek.rs

```diff
@@ -20,8 +20,10 @@ use mz_expr::{CollectionPlan, ResultSpec};
 use mz_ore::task;
 use mz_ore::tracing::OpenTelemetryContext;
 use mz_repr::explain::{ExprHumanizerExt, TransientItem};
+use mz_repr::optimize::OverrideFrom;
 use mz_repr::{Datum, GlobalId, Row, RowArena, Timestamp};
-use mz_sql::catalog::{CatalogCluster, SessionCatalog};
+use mz_sql::catalog::CatalogCluster;
+use mz_sql::catalog::SessionCatalog;
```

Comment:
Did you mean to pull these apart?

Reply:
No, my editor messed this up. Will fix this.